x/storage/types/message.go

```diff
@@ -1222,14 +1223,25 @@ func (msg *MsgMirrorBucket) ValidateBasic() error {
 		return errors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
 
+	if msg.Id.GT(sdk.NewUint(0)) {
```

Comment:
Can we enforce that the `BucketName` is empty of `ID` is greater than 0? 
So we can make the msg clean.

Reply:
Added the validation now.